README.rst

```diff
@@ -543,6 +543,11 @@ Choose from the list of available rules:
   Add curly braces to indirect variables to make them clear to understand.
   Requires PHP >= 7.0.
 
+* **explicit_string_variable**
```

Comment:
I would add it to local `.php_cs.dist` file and then merge PR, wdyt @Slamdunk ?

Reply:
Done

Reply:
Done